File: src/main/java/graph/edge/SimplexChannel.java
package graph.edge;

import graph.vertex.Vertex;

public class SimplexChannel extends AbstractEdge {
    public SimplexChannel(Vertex fromNode, Vertex toNode, int weight, double errors) {
        super(fromNode, toNode, weight, errors);
        connect();
    }

    @Override
    protected String getDirection() {
        return "-->";
    }
}

File: src/main/java/mvc/model/channel/SimplexChannel.kt
package mvc.model.channel

import mvc.model.node.Node

class SimplexChannel(fromNode: Node, toNode: Node, weight: Int, errors: Double) : AbstractChannel(fromNode, toNode, weight, errors) {

    override val direction: String
        get() = "-->"

    init {
        connect()
    }
}
